### PR TITLE
[mail] add actions with optional headers

### PR DESCRIPTION
**Pull request by J-N-K**
Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

File: bundles/org.smarthomej.binding.mail/README.md [hunk 1]
```diff
@@ -149,9 +156,9 @@ success = mailActions.sendMail("mail1@example.com, mail2@example.com", "Test sub
 ```
 import java.util.List
 
-val List<String> attachmentUrlList = newArrayList(
+val List<String> attachmentUrlList = new ArrayList(
```
Comment (cweitkamp) [inline bundles/org.smarthomej.binding.mail/README.md:159]: Does this work without importing it? Can we use `List.of(...)` method?

File: bundles/org.smarthomej.binding.mail/src/main/resources/OH-INF/i18n/mail.properties [hunk 2]
```diff
@@ -16,3 +19,6 @@ sendHTMLAttachmentMessageActionDescription = Sends a HTML mail with an URL attac
 
 sendHTMLAttachmentsMessageActionLabel = send a HTML mail with several attachments
 sendHTMLAttachmentsMessageActionDescription = Sends a HTML mail with several URL attachments.
+
+sendComplexHtmlMessageActionLabel = send a html mail with several attachments and/or additional headers
```
Comment (cweitkamp) [inline bundles/org.smarthomej.binding.mail/src/main/resources/OH-INF/i18n/mail.properties]:
```suggestion
sendComplexHtmlMessageActionLabel = send an HTML mail with several attachments and/or additional headers
```

In general we should fix the spelling in this file `a HTML` to `an HTML`.

**Review by cweitkamp — DISMISSED** [review]
LGTM. Only minor remarks.

File: bundles/org.smarthomej.binding.mail/README.md (hunk 1 above)
Comment (J-N-K) [reply bundles/org.smarthomej.binding.mail/README.md:159]: Good question. I don't know (both questions). I guess we should leave it like it is for the moment, I just noticed the missing space and added it.

**Review by cweitkamp — APPROVED** [review]

**Pull request merged**